redis/src/main/java/org/apache/zeppelin/redis/command/RedisCommand.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.redis.command;
+
+import org.apache.zeppelin.interpreter.InterpreterResult;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.JedisPool;
+import redis.clients.jedis.exceptions.JedisException;
+
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.ERROR;
+import static org.apache.zeppelin.interpreter.InterpreterResult.Code.SUCCESS;
+
+/**
+ * Redis BLPOPCommand
+ */
+public abstract class RedisCommand {
+
+    private JedisPool clientPool;
+
+    public RedisCommand(JedisPool clientPool) {
+        this.clientPool = clientPool;
+    }
+
+    private Jedis borrowClient() {
+        return clientPool.getResource();
+    }
+
+    private void releaseClient(Jedis client) {
+        client.close();
+    }
+
+    public InterpreterResult execute(String[] tokens) {
+        Jedis client = borrowClient();
+        try {
+            return new InterpreterResult(SUCCESS, execute(client, tokens));
+        } catch (JedisException exception) {
+            return new InterpreterResult(ERROR, exception.getMessage());
```

Comment:
What do you think about adding a `LOG.error` here? something like
`LOG.error("Couldn't execute redis command {}", token[0], e);`